src/persistent_merkle_tree/Node.zig

```diff
@@ -205,6 +205,18 @@ pub const Pool = struct {
         return self.createUnsafe(self.nodes.items(.state));
     }
 
+    /// Returns the number of nodes currently in use (not free)
+    pub fn getNodesInUse(self: *Pool) usize {
```

Comment:
this is not supposed for application, it's for tests only where we want to make sure there is no orphaned nodes